utils/python/CIME/case_setup.py

```diff
@@ -84,7 +84,7 @@ def case_setup(caseroot, clean=False, test_mode=False, reset=False):
 
     cimeroot = os.environ["CIMEROOT"]
 
-    case = Case()
+    case = Case(caseroot)
```

Comment:
I had already put in a pull request my work around. I am happy to do the set_trace commands and track this down - but I will not get a chance to do this before Monday.